### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by oliverfoster**
Changed:
* coding style to be more inline with adapt
* size selectors to classes

Added:
* ``onMediaQuery`` selectors
* ``_isEnabled``, ``_isDynamic`` attributes
* schema

### CODE REVIEW

File: js/adapt-hide.js [hunk 1]
```diff
@@ -1,54 +1,94 @@
...
 
-	Adapt.on("device:changed", checkStates);
+        onPostRender: function(view) {
+            var model = view.model;
+            var config = model.get("_hide");
 
-	Adapt.on("remove", function() {
-		managedViews.length = 0;
-	});
+            if (config) console.log(config);
```
Comment (moloko) [inline js/adapt-hide.js]: better to use `Adapt.log` here?

Comment (oliverfoster) [reply js/adapt-hide.js]: ahh, i hadn't meant to leave that in. i shall remove. ta

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "adapt-hide",
-	"version": "0.1.1",
+	"version": "1.1.1",
```
Comment (tomgreenfield) [inline bower.json]: `0.1.1` to `1.1.1`?

Comment (oliverfoster) [reply bower.json]: yea, i left that for you. what do you want?

**Pull request merged**